Extensions/old_stats.js

```diff
@@ -1,108 +1,48 @@
 //* TITLE Old Stats **//
-//* VERSION 0.2.2 **//
+//* VERSION 0.3.0 **//
 //* DESCRIPTION  **//
 //* DEVELOPER STUDIOXENIX **//
 //* FRAME false **//
 //* BETA false **//
 
 XKit.extensions.old_stats = new Object({
 
-	running: false,
+    running: false,
+
+    preferences: {
+        "sep-0": {
+            text: "Old Stats is now Old Sidebar",
+            type: "separator"
+        },
+    },
+
+    run: function() {
+        this.running = true;
+        XKit.install("old_sidebar", function(mdata) {
```

Comment:
we should probably not try to install old_sidebar if old_sidebar is already installed (unless install checks for this? I can't imagine it does...)